### PR TITLE
BigMac-Shield alerts migration

### PR DESCRIPTION
**Pull request by ssyno**
Before adding a new alerting rule into this repository you should consider creating an SLO rules instead.
SLO helps you both increase the quality of your monitoring and reduce the alert noise.

* How to create a SLO rule: https://github.com/giantswarm/sloth-rules#how-to-create-a-slo
* Documentation: https://intranet.giantswarm.io/docs/monitoring/slo-alerting/

---
Towards: https://github.com/giantswarm/...

This PR ...

### Checklist

- [x] Update CHANGELOG.md
- [x] Add [Unit tests](https://github.com/giantswarm/prometheus-rules/#testing)
- [x] Follow [Alert structure](https://github.com/giantswarm/prometheus-rules/#how-alerts-are-structured)
- [ ] Consider [creating a dashboard](https://docs.giantswarm.io/getting-started/observability/grafana/custom-dashboards/) ([guidelines](https://intranet.giantswarm.io/docs/product/ux/guidelines/dashboards/)) (if it does not exist already) to help oncallers monitor the status of the issue.
- [ ] Request review from oncall area, as well as team (e.g: `oncall-kaas-cloud` GitHub group).

### CODE REVIEW

File: helm/prometheus-rules/templates/kaas/bigmac/alerting-rules/cert-manager.rules.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@ spec:
         area: kaas
         cancel_if_outside_working_hours: {{ include "workingHoursOnly" . }}
         severity: notify
-        team: bigmac
+        team: shiel
```
Comment (QuentinBisson) [inline helm/prometheus-rules/templates/kaas/bigmac/alerting-rules/cert-manager.rules.yml:26]: You mean shield right ? 😁

File: helm/prometheus-rules/templates/platform/atlas/alerting-rules/deployment.management-cluster.rules.yml [hunk 2]
```diff
@@ -132,7 +132,7 @@ spec:
         severity: page
         team: cabbage
         topic: managementcluster
-    - alert: DeploymentNotSatisfiedBigMac
+    - alert: DeploymentNotSatisfiedsShield
```
Comment (QuentinBisson) [inline helm/prometheus-rules/templates/platform/atlas/alerting-rules/deployment.management-cluster.rules.yml:135]:
```suggestion
    - alert: DeploymentNotSatisfiedShield
```

File: test/tests/providers/vintage/aws/platform/shield/alerting-rules/cert-manager.rules.test.yml [hunk 3]
```diff
@@ -32,7 +32,7 @@ tests:
               installation: gollem
               service_priority: highest
               severity: page
-              team: bigmac
+              team: shiel
```
Comment (QuentinBisson) [inline test/tests/providers/vintage/aws/platform/shield/alerting-rules/cert-manager.rules.test.yml:35]:
```suggestion
              team: shield
```

File: helm/prometheus-rules/templates/kaas/bigmac/alerting-rules/cert-manager.rules.yml (hunk 1 above)
Comment (ssyno) [reply helm/prometheus-rules/templates/kaas/bigmac/alerting-rules/cert-manager.rules.yml:26]: Damn fixing

**Comment by QuentinBisson** [comment]
Could you also move the folders? You'll probably have to replace kaas with platform

File: helm/prometheus-rules/templates/platform/shield/alerting-rules/cert-manager.rules.yml [hunk 4]
```diff
@@ -23,7 +23,7 @@ spec:
         area: kaas
         cancel_if_outside_working_hours: {{ include "workingHoursOnly" . }}
         severity: notify
-        team: bigmac
+        team: shiel
```
Comment (QuentinBisson) [inline helm/prometheus-rules/templates/platform/shield/alerting-rules/cert-manager.rules.yml:26]:
```suggestion
        team: shield
```

File: helm/prometheus-rules/templates/platform/shield/alerting-rules/cert-manager.rules.yml [hunk 5]
```diff
@@ -38,7 +38,7 @@ spec:
         cancel_if_cluster_status_deleting: "true"
         cancel_if_kubelet_down: "true"
         severity: page
-        team: bigmac
+        team: shiel
```
Comment (QuentinBisson) [inline helm/prometheus-rules/templates/platform/shield/alerting-rules/cert-manager.rules.yml:41]:
```suggestion
        team: shield
```

File: helm/prometheus-rules/templates/platform/atlas/alerting-rules/deployment.management-cluster.rules.yml [hunk 6]
```diff
@@ -132,7 +132,7 @@ spec:
         severity: page
         team: cabbage
         topic: managementcluster
-    - alert: DeploymentNotSatisfiedBigMac
+    - alert: DeploymentNotSatisfiedShield
```
Comment (QuentinBisson) [inline helm/prometheus-rules/templates/platform/atlas/alerting-rules/deployment.management-cluster.rules.yml:135]: Can you also switch area: kaas label to area: platform?

Comment (stone-z) [reply helm/prometheus-rules/templates/platform/atlas/alerting-rules/deployment.management-cluster.rules.yml:135]: An alert with this name already exists https://github.com/giantswarm/prometheus-rules/pull/1377/files#diff-5ea599b46b0488bf1d9fdd9264ee35236e05930d4a7747105a33e4813231e768R57

Comment (QuentinBisson) [reply helm/prometheus-rules/templates/platform/atlas/alerting-rules/deployment.management-cluster.rules.yml:135]: That is fine but we can improve by merging them yes

**Pull request merged**